core/src/main/java/ee/tlu/evkk/core/service/TextService.java

```diff
@@ -45,6 +48,183 @@ public List<TextWithProperties> search(Pageable pageable, String[] korpus, Strin
     return texts.stream().map(text -> toTextWithProperties(text, textPropertiesByTextId)).collect(Collectors.toUnmodifiableList());
   }
 
+  public String[] translateWordType(String[] tekst) {
+    String[] returnText = stream(tekst).toArray(String[]::new);
```

Comment:
Kui juba streamiga lahendatud, siis võiks map funktsiooni kasutada. https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#map-java.util.function.Function-

Reply:
@mikktarvas Kasutan streami suvaliselt ühena mitmest variandist, millega on võimalik deep copy teha. Kuna ma sisu muuta ei taha, siis kuidas ma mapi kasutama peaksin? IDE ütleb ka, et see on sellisel juhul reduntant. Googeldades leidsin ainult soovitusi, mis teevad nii nagu hetkel on (ja ka docsis soovitatud), nt siin: https://stackoverflow.com/questions/23079003/how-to-convert-a-java-8-stream-to-an-array

Reply:
Sisu muudetakse real 54. Funktsiooni saaks lihtsustada:
`  public String[] translateWordType(String[] tekst) {
    return Arrays.stream(tekst).map(elem -> wordTypes.get(elem)).toArray(String[]::new);
  }`

Reply:
Minu viga, täitsa õige. Nüüd see tehtud ja saab uuesti üle vaadata PRi.